tests/functional/pyocf/types/volume.py

```diff
@@ -341,6 +344,71 @@ def new_io(
         )
         return Io.from_pointer(io)
 
+    def sync_io(
+        self,
+        queue,
+        address: int,
+        data: Data,
+        direction: IoDir,
+        io_class=0,
+        flags=0,
+        submit_func=Sync.submit,
+    ):
+        assert address % 512 == 0
+        assert data.size % 512 == 0
+
+        io = self.new_io(queue, address, data.size, direction, io_class, flags)
+        io.set_data(data)
+        completion = submit_func(Sync(io))
+
+        assert int(completion.results["err"]) == 0
+
+    def write_sync_4k(
+        self,
+        queue: Queue,
+        address: int,
+        data: Union[bytes, Data],
+        mode: WriteMode,
+        io_class=0,
+        flags=0,
+    ):
+        if mode not in list(WriteMode):
+            raise ValueError(f"illegal write mode: {mode}")
+
+        size = len(data)
+
+        address_4k = (address // 4096) * 4096
+
+        end_address_4k = ((address + size + 4095) // 4096) * 4096
+        size_4k = end_address_4k - address_4k
```

Comment:
use util.align_to_4k?

Reply:
Actually, I wanted to get rid of that function. just forgot to do so.